Description: Use Debian's copies of the GPL licenses in the documentation.
 The ones that ship with Macaulay2 are gzipped by dh_compress.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2020-05-21

--- a/M2/Macaulay2/packages/Macaulay2Doc/overview3.m2
+++ b/M2/Macaulay2/packages/Macaulay2Doc/overview3.m2
@@ -6,7 +6,7 @@
      Key => "COPYING-GPL-2",
      Headline => "GNU General Public License, version 2",
      if prefixDirectory =!= null
-     then PRE get(prefixDirectory | currentLayout#"docdir" | "COPYING-GPL-2")
+     then PRE get("/usr/share/common-licenses/GPL-2")
      else (
 	  stderr << "warning: can't locate file \"COPYING-GPL-2\"" << endl;
 	  SPAN {"See the GNU GENERAL PUBLIC LICENSE, Version 2, June 1991, available at ", HREF "http://www.gnu.org/licenses/gpl-2.0.txt", "."}
@@ -17,7 +17,7 @@
      Key => "COPYING-GPL-3",
      Headline => "GNU General Public License, version 3",
      if prefixDirectory =!= null
-     then PRE get(prefixDirectory | currentLayout#"docdir" | "COPYING-GPL-3")
+     then PRE get("/usr/share/common-licenses/GPL-3")
      else (
 	  stderr << "warning: can't locate file \"COPYING-GPL-3\"" << endl;
 	  SPAN {"See the GNU GENERAL PUBLIC LICENSE, Version 3, 29 June 2007, available at ", HREF "http://www.gnu.org/licenses/gpl-3.0.txt", "."}
